### PR TITLE
Remove services folder

### PR DESCRIPTION
**Pull request by chao-xian**
This PR follows on from the restructuring of the code in #654. The code in `services` needed a little bit of refactoring before we could move the files, so was deemed out of scope for #654 as that was a lift and shift but gave us the basis for this PR.

The `ItemsService` file has been removed as it added an additional level of abstraction that appeared to be unnecessary.

### CODE REVIEW

**Comment by chao-xian** [comment]
[(2) Refactor and move code in Services folder into the new app structure](https://trello.com/c/kddke6Jo/247-2-refactor-and-move-code-in-services-folder-into-the-new-app-structure)

File: app/etl/item/clients/google_analytics.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require 'google/apis/analyticsreporting_v4'
+require 'googleauth'
+
+class Item::Clients::GoogleAnalytics
```
Comment (pmanrubia) [inline app/etl/item/clients/google_analytics.rb]: What about moving this client to ETL/GA folder?

File: app/etl/item/clients/publishing_api.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
+require 'gds_api/publishing_api_v2'
+
+class Item::Clients::PublishingAPI
```
Comment (pmanrubia) [inline app/etl/item/clients/publishing_api.rb]: `PublishingAPI` is used in the `items` process. What are your thoughts about moving it there?

File: app/etl/item/clients/publishing_api.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  PER_PAGE = 50
```
Comment (pmanrubia) [inline app/etl/item/clients/publishing_api.rb]: If we have this constant `PER_PAGE`, do we also need the instance variable `@per_page`?

File: app/etl/item/clients/publishing_api.rb (hunk 2 above)
Comment (chao-xian) [reply app/etl/item/clients/publishing_api.rb]: Fixed in 6f512e2ec47733f83831527af491fcaac224590e

File: app/etl/item/clients/publishing_api.rb (hunk 3 above)
Comment (chao-xian) [reply app/etl/item/clients/publishing_api.rb]: Fixed in d625cb032655d13beb3378a33dca9b12cd702d81

**Comment by chao-xian** [comment]
Thanks @pmanrubia - have addressed your comments.

**Pull request merged**